controllers/storageclient_controller.go

```diff
@@ -526,98 +456,63 @@ func (s *StorageClientReconciler) reconcileClientStatusReporterJob(instance *v1a
 	return reconcile.Result{}, nil
 }
 
-func (s *StorageClientReconciler) list(obj client.ObjectList, listOptions ...client.ListOption) error {
-	return s.Client.List(s.ctx, obj, listOptions...)
+func (r *StorageClientReconciler) list(obj client.ObjectList, listOptions ...client.ListOption) error {
+	return r.Client.List(r.ctx, obj, listOptions...)
 }
 
-func (s *StorageClientReconciler) reconcileDefaultStorageClaims(instance *v1alpha1.StorageClient) error {
-
-	if instance.GetAnnotations()[defaultClaimsProcessedAnnotationKey] == "true" {
-		// we already processed default claims for this client
+func (r *StorageClientReconciler) reconcileBlockStorageClaim() error {
+	if r.storageClient.GetAnnotations()[storageClaimProcessedAnnotationKey] == "true" {
+		// we already processed claim creation for this client
 		return nil
 	}
```

Comment:
The annotation concern is added at a upper level. I would expect the guard to be outside of the reconciler function and not inside it.

Reply:
makes sense, will do.

Reply:
done.